### PR TITLE
webapp/cas: use common config

### PR DESCRIPTION
**Pull request by tharvik**
* cas was using two config, which got out of sync
* now, it uses the same config, so adding a parsing for it

### CODE REVIEW

File: webapp/src/app/api/v0/cas/config.service.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+import { InstanceID } from "@dedis/cothority/byzcoin";
+
+export class Action {
+    constructor(
+        readonly darc: InstanceID,
+        readonly coin: InstanceID,
+    ) {}
+}
+
+const getField = (obj: object, name: string): unknown => {
```
Comment (ineiti) [inline webapp/src/app/api/v0/cas/config.service.ts]: Why are these defined as `const` and not as `function`?

File: webapp/src/app/api/v0/cas/config.service.ts [hunk 2]
```diff
@@ -0,0 +1,147 @@
+import crypto from "crypto";
+import { List, Map } from "immutable";
+import toml from "toml";
+
+import { Injectable } from "@angular/core";
+
+import { InstanceID } from "@dedis/cothority/byzcoin";
+
+export class Action {
```
Comment (ineiti) [inline webapp/src/app/api/v0/cas/config.service.ts]: I'd take a small comment here (and other exported symbols) for two kind of persons:
- you in 2 months
- everybody else

File: webapp/src/app/api/v0/cas/login/login.component.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import crypto from "crypto-browserify";
+import crypto from "crypto";
```
Comment (ineiti) [inline webapp/src/app/api/v0/cas/login/login.component.ts]: Please use `crypto-browserify` for the moment. Else we have two different sources of crypto, and I don't want to chase those bugs.

File: webapp/src/app/api/v0/cas/config.service.ts (hunk 1 above)
Comment (tharvik) [reply webapp/src/app/api/v0/cas/config.service.ts]: Cruft from a previous iteration, fixing.

**Pull request merged**